.script/ValidDomainsVerticals.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This config file live in a same folder as script or can we keep at a separate place. I am not sure how it was done for other testcases

Reply:
placed it in separate folder.